cli/cli/out/file_logger.go

```diff
@@ -0,0 +1,68 @@
+package out
+
+import (
+	"github.com/kurtosis-tech/kurtosis/cli/cli/helpers/host_machine_directories"
+	"github.com/sirupsen/logrus"
+	"os"
+)
+
+// FileLogger this logger will only log to a file
+// TODO: In commands like inspect we use out.PrintOutLn - will need to add this fileLogger
+//  to print commands' output as well.
+var FileLogger = logrus.New()
+
+func InitFileLogger() {
+	// kurtosis-cli.log can be found in the same directory as kurtosis-config.yml
+	filePath, err := host_machine_directories.GetKurtosisCliLogsFilePath()
+	if err != nil {
+		panic(err)
```

Comment:
why are you adding panic here instead of adding stacktrace.Propagate ?

Reply:
good catch; will fix this.